### PR TITLE
feat: [sveltekit-pod] 2. Add routing and create homepage

### LINKED ISSUE #378: [sveltekit-pod] 2. Add routing and create homepage
# Background
We want to demonstrate basic routing as part of each kit. To do so, we want a homepage that directs to all the examples we've created thus far.

# Acceptance
- [ ] Add and configure routing
- [ ] Create a basic homepage that lives at `/`
- [ ] Route the counter component to `/counter`
- [ ] Route the API component to `/api-example`

# Note:
- The mock was taken from a different kit. Change the text to be align with the current kit

# Mock
image

### PR DESCRIPTION
**Pull request by ihardz**
Resolves #378

> 
> # Background
> We want to demonstrate basic routing as part of each kit. To do so, we want a homepage that directs to all the examples we've created thus far.
> 
> # Acceptance
> - [x] Add and configure routing
> - [x] Create a basic homepage that lives at `/`
> - [x] Route the counter component to `/counter`
> - [x] Route the API component to `/api-example`
> 
> # Note:
> - The mock was taken from a different kit. Change the text to be align with the current kit
> 
> # Mock
> image
> 

Result: 
image

### CODE REVIEW

File: starters/svelte-kit-scss/src/routes/(examples)/api-example/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	import Counter from '$lib/components/Counter.svelte';
+	import PageLayout from '$lib/components/PageLayout.svelte';
+</script>
+
+<PageLayout>
+	<span slot="heading">Fetch Data From API</span>
+	<div>
+		Fetch component works
+	</div>
+</PageLayout>
```
Comment (flakolefluk) [inline starters/svelte-kit-scss/src/routes/(examples)/api-example/+page.svelte:11]: I'd try to have a header/main/footer on each page (if possible keep the in the layout)
but:
```suggestion
<Header>
	<h1>Fetch Data From API</span>
</Header>
<main>
	<div>
		Fetch component works
	</div>
</main>
```

File: starters/svelte-kit-scss/src/routes/(examples)/counter/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<script>
+	import Counter from '$lib/components/Counter.svelte';
+	import PageLayout from '$lib/components/PageLayout.svelte';
+</script>
+
+<PageLayout>
+	<span slot="heading">Increment, Decrement and Reset Button Example</span>
+	<Counter />
+</PageLayout>
```
Comment (flakolefluk) [inline starters/svelte-kit-scss/src/routes/(examples)/counter/+page.svelte:9]: Same idea as before

File: starters/svelte-kit-scss/src/routes/(examples)/counter/+page.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import { dev } from '$app/environment';
+// import { dev } from '$app/environment';
 
 // we don't need any JS on this page, though we'll load
 // it in dev so that we get hot module replacement
-export const csr = dev;
+// export const csr = dev;
```
Comment (flakolefluk) [inline starters/svelte-kit-scss/src/routes/(examples)/counter/+page.ts:5]: why are these commented out?

File: starters/svelte-kit-scss/src/routes/+page.svelte [hunk 4]
```diff
@@ -1,54 +1,13 @@
...
-	}
-</style>
+	<ul>
+		<li >
+			<a href="/counter">See Counter example component</a>
+		</li>
+		<li >
+			<a href="/api-example">See Fetch example component</a>
+		</li>
+	</ul>
```
Comment (flakolefluk) [inline starters/svelte-kit-scss/src/routes/+page.svelte:12]: are these part of the header?

**Review by flakolefluk — CHANGES_REQUESTED** [review]
Happy to pair on this tomorrow

**Review by iansamz — APPROVED** [review]
LGTM

**Review by iansamz — CHANGES_REQUESTED** [review]
If not being used please remove
`starters/svelte-kit-scss/src/routes/(examples)/api-example/words.server.ts`

**Comment by iansamz** [comment]
Please align design of home page to mock

**Review by flakolefluk — APPROVED** [review]
Looks good to me, I think this aligns better with the sveltekit structure, and keeps it aligned with other starters in terms of markup.

**Review by iansamz — APPROVED** [review]
LGTM 👍🏾

**Pull request merged**